frontend/src/component/Summary.js

```diff
@@ -0,0 +1,45 @@
+import React, { Fragment } from "react"
+
+const Summary = ({ charts }) => {
+  if (!charts || charts.length === 0) {
+    return null
+  }
+  const chartsByTitle = charts.reduce((obj, chart) => {
+    obj[chart.title] = chart
+    return obj
+  }, {})
+  // TODO(juliaogris): find a more generic way of expressing the following
```

Comment:
Well the idea/hope was to "generify" this leaderboard.
People / users could choose what to display in their leaderboard (therefore also the chartID in the backend code).
E.g. somebody might be interested in issue metrics or changed lines, but not so much in reviews or so.... 
Maybe that's all stupid. I've removed the TODO. If I ever get to extend this thing in this way it will require a rework here anyway.
Done.

Reply:
i had a bit of an idea what this todo was about, but i couldn't find a way to simplify it to one line. the alternative is `TODO(#x): Make charts generic` or something simple like that, and have the details in issue #x. now I think about it, perhaps the text of the todo should be the title of the issue.